### ISSUE TITLE
Floating Ben

### ISSUE THREAD

**Issue by araeden** [smealum/ludum34#11]
Sometimes if you rotate out the layer Ben is supposed to land on, he lands in mid-air. It doesn't break the game, just looks weird.

**Comment by smealum** [comment]
yeah noticed that during the last few hours of the stream (right after i implemented autopilot) ! it's unfortunate that i couldnt to fix it in time. thanks for reporting
